### PR TITLE
testing only

### PR DESCRIPTION
**Pull request by alexbatashev**

### CODE REVIEW

File: .github/workflows/sycl_linux_build_and_test.yml [hunk 1]
```diff
@@ -326,3 +331,49 @@ jobs:
...
+      image: ${{ fromJSON(needs.configure.outputs.params).build_image }}
+      options: -u 1001:1001
+    steps:
+    - run: cp -r /actions .
+    - name: Register cleanup after job is finished
+      uses: ./actions/cleanup
+    - name: Download compiler toolchain
+      uses: actions/download-artifact@v2
+      with:
+        name: sycl_cts_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
```
Comment (alexbatashev) [inline .github/workflows/sycl_linux_build_and_test.yml:349]:
```suggestion
        name: sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
```

File: .github/workflows/sycl_linux_build_and_test.yml [hunk 2]
```diff
@@ -326,3 +331,49 @@ jobs:
...
+    - name: Download compiler toolchain
+      uses: actions/download-artifact@v2
+      with:
+        name: sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
+    - name: Extract SYCL toolchain
+      shell: bash
+      run: |
+        mkdir toolchain
+        tar -xf sycl_cts_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }} -C toolchain
+        rm -f sycl_cts_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
```
Comment (alexbatashev) [inline .github/workflows/sycl_linux_build_and_test.yml:355]:
```suggestion
        tar -xf sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }} -C toolchain
        rm -f sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
```

File: .github/workflows/sycl_linux_build_and_test.yml [hunk 3]
```diff
@@ -326,3 +331,49 @@ jobs:
...
+      uses: ./actions/cleanup
+    - name: Download compiler toolchain
+      uses: actions/download-artifact@v2
+      with:
+        name: sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
+    - name: Extract SYCL toolchain
+      shell: bash
+      run: |
+        mkdir toolchain
+        tar -xf sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }} -C toolchain
```
Comment (alexbatashev) [inline .github/workflows/sycl_linux_build_and_test.yml:354]:
```suggestion
        tar -xf sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}.tar.xz -C toolchain
```

**Comment by bader** [comment]
Why don't you use a container with re-built compiler to validate testing scripts?

**Comment by alexbatashev** [comment]
> Why don't you use a container with re-built compiler to validate testing scripts?
> Obviously this allows us to skip Build+LIT step.

I wanted to see how much time it’d take to go through whole pipeline with all downloads/uploads. Didn’t realize it would take this many tries.

**Pull request closed without merging**